### PR TITLE
Updated weight normalization and validation set normalization

### PR DESCRIPTION
**Pull request by sijoerger**

### CODE REVIEW

**Review by stwunsch — APPROVED** [review]
Das lwtnn submodule ist gar nicth verwendet. Kannst du das noch rauswerfen?

**Comment by stwunsch** [comment]
Oh sry, missed that!

**Comment by stwunsch** [comment]
Simon, kannst wieder den comimt reverten? :D

**Comment by stwunsch** [comment]
Du kannst einfach den commit reverten, dann rebasen und die commits squashen.

File: training/keras_training.py [hunk 1]
```diff
@@ -206,17 +206,34 @@ def balancedBatchGenerator(batch_size):
...
+                [w_test[testIndexDict[class_]] * (len(x_test) / sum_class[class_]) for class_ in classes])
+
+            return x_collect, y_collect, w_collect
+
+        x_test, y_test, w_test = calculateValidationWeights(x_test, y_test, w_test)
+
+        history = model.fit_generator(
             balancedBatchGenerator(batch_size=batch_size),
-            steps_per_epoch=10,
+            steps_per_epoch=len(x_train) // batch_size,
```
Comment (stwunsch) [inline training/keras_training.py:231]: Funktioniert das in python 2?

Comment (sijoerger) [reply training/keras_training.py:231]: Uh, das standard repo benutzt doch python 2.7 oder? Also ja?

Comment (stwunsch) [reply training/keras_training.py:231]: Ok alles klar, das wollte ich wissen :)

**Comment by stwunsch** [comment]
@sijoerger (@ArturAkh ) Lasst morgen im Meeting die Slides diskutieren und dann mergen wir das rein!
